recipe/meta.yaml

```diff
@@ -74,11 +72,12 @@ test:
     - if not exist "%PREFIX%\\Scripts\\nionui-tool.exe" exit 1  # [win]
     - if not exist "%PREFIX%\\Scripts\\NionUILauncher\\NionUILauncher.exe" exit 1  # [win]
     - if not exist "%PREFIX%\\Scripts\\NionUILauncher\\bootstrap.py" exit 1  # [win]
-    - /usr/bin/sudo -n yum install -y xorg-x11-server-Xvfb  # [linux]
     - python -m pip install nionui_app_test/
     - nionui nionui_app.test_ack | grep ACK  # [osx]
     - nionui nionui_app.test_ack | findstr /c:"ACK"  # [win]
-    - xvfb-run -a nionui nionui_app.test_ack | grep ACK  # [linux]
+    # disable linux tests until xvfb-run reliably exits after running the app
+    # - /usr/bin/sudo -n yum install -y xorg-x11-server-Xvfb  # [linux]
+    # - xvfb-run -a nionui nionui_app.test_ack  # [linux]
```

Comment:
Raised issue ( https://github.com/conda-forge/nionui-tool-feedstock/issues/51 ) with suggestions on how to better handle this need